geoapps_utils/geophysical_systems.py

```diff
@@ -0,0 +1,3889 @@
+            "bird_offset": [0, 0, 0],
+            "normalization": [3.1416, 1e-12],
+            "tx_specs": {"type": "CircularLoop", "a": 1.0, "I": 1.0},
+            "data_type": "dBzdt",
+        },
+        "Zonge 8Hz": {
+            "type": "time",
+            "flag": "CH",
+            "channel_start_index": 6,
+            "channels": {
+                "1": 8.620e-05,
+                "2": 1.168e-04,
+                "3": 1.473e-04,
+                "4": 1.778e-04,
+                "5": 2.083e-04,
+                "6": 2.388e-04,
+                "7": 2.846e-04,
+                "8": 3.456e-04,
+                "9": 4.067e-04,
+                "10": 4.830e-04,
+                "11": 5.745e-04,
+                "12": 6.966e-04,
+                "13": 8.644e-04,
+                "14": 1.063e-03,
+                "15": 1.307e-03,
+                "16": 1.612e-03,
+                "17": 2.009e-03,
+                "18": 2.528e-03,
+                "19": 3.169e-03,
+                "20": 3.962e-03,
+                "21": 4.954e-03,
+                "22": 6.220e-03,
+                "23": 7.822e-03,
+                "24": 9.806e-03,
+                "25": 1.231e-02,
+                "26": 1.548e-02,
+                "27": 1.946e-02,
+                "28": 2.449e-02,
+            },
+            "uncertainty": [
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+                [0.05, 1e-0],
+            ],
+            "waveform": [
+                [-3.0225e-02, 1.0000e-08],
+                [-2.5225e-02, 5.0000e-01],
+                [-2.0225e-02, 1.0000e00],
+                [-1.5225e-02, 1.0000e00],
+                [-1.0225e-02, 1.0000e00],
+                [-5.2250e-03, 1.0000e00],
+                [-2.2500e-04, 1.0000e00],
+                [-2.0000e-04, 1.0000e00],
+                [-1.7500e-04, 8.7500e-01],
+                [-1.5000e-04, 7.5000e-01],
+                [-1.2500e-04, 6.2500e-01],
+                [-1.0000e-04, 5.0000e-01],
+                [-7.5000e-05, 3.7500e-01],
+                [-5.0000e-05, 2.5000e-01],
+                [-2.5000e-05, 1.2500e-01],
+                [-0.0000e00, 0.0000e00],
```

Comment:
**question (readability):** could this be generated with increment of `2.5e-05` instead of hard-coding it all?